requirements.txt

```diff
@@ -7,7 +7,9 @@ botocore==1.19.25
 Brotli==1.0.9
 certifi==2020.11.8
 cffi==1.14.4
+chardet==3.0.4
```

Comment:
🤔

Reply:
<img width="706" alt="Screen Shot 2020-12-01 at 10 31 23 PM" src="https://user-images.githubusercontent.com/43714531/100824925-fe02de80-3424-11eb-8845-33ceb35979c8.png">
not pretty sure why it is there.... do you want me to remove it? @gzhami